src/models/modelTests/DRBResult.test.ts

```diff
@@ -0,0 +1,33 @@
+import { drbRawData } from "../../../__test__/fixtures/drbResultsFixtures"
+import type { DRBWork } from "../../types/drbTypes"
+
+import DRBResult from "../DRBResult"
+
+const works = drbRawData?.data?.works
+
+describe("DRBResult model", () => {
+  const model = new DRBResult(works[0] as DRBWork)
+
+  it("get basic DRB information for a work", () => {
+    expect(model.id).toEqual("84306f57-3285-403a-bff4-3d8286d98183")
+    expect(model.title).toEqual("Anatomy of the cat")
+    // Authors still contains all the additional properties but are
+    // not used and not defined in the model or Author interface
+    expect(model.authors).toEqual([
+      { name: "Reighard, Jacob (1861-)", lcnaf: "", primary: "true", viaf: "" },
```

Comment:
I noticed that even though the `Author` interface is defined as `{ name: string }`, all these values are still passed to the object. This is also true for the general model object. Perhaps we can be pickier and `Pick<....>` only the properties we want from the API data response.